eng/jacoco-test-coverage/pom.xml

```diff
@@ -72,13 +72,13 @@
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-keyvault-keys</artifactId>
-      <version>4.0.0-preview.6</version> <!-- {x-version-update;com.azure:azure-keyvault-keys;current} -->
+      <artifactId>azure-security-keyvault-keys</artifactId>
+      <version>4.0.0-preview.5</version> <!-- {x-version-update;com.azure:azure-security-keyvault-keys;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-keyvault-secrets</artifactId>
-      <version>4.0.0-preview.6</version> <!-- {x-version-update;com.azure:azure-keyvault-secrets;current} -->
+      <artifactId>azure-security-keyvault-secrets</artifactId>
+      <version>4.0.0-preview.5</version> <!-- {x-version-update;com.azure:azure-security-keyvault-secrets;current} -->
```

Comment:
This has gone from preview.6 to preview.5. Was that intentional?

Reply:
it was bumped twice.
1. when Storage was possibly re-releasing KV (but didn't happen)
2. when Jim's script was ran. (bumped version again)
So, I reverted it back to P5, as that's what KV next release will be.
Already discussed with @srnagar